docs/docs/0.4.0/policies/fault-injection.md

```diff
@@ -0,0 +1,92 @@
+# Fault Injection
+
+`FaultInjection` policy helps you to test your microservices against resiliency. Kuma provides 3 different types of failures that could be imitated in your environment. 
+These faults are [Delay](#delay), [Abort](#abort) and [ResponseBandwidth](#responsebandwidth-limit) limit.
+
+On Universal:
+
+```yaml
+type: FaultInjection
+mesh: default
+name: fi1
+sources:
+    - match:
+        service: frontend
+        version: "0.1"
```

Comment:
We've got required http tag, right? We should add it to examples, otherwise they are not valid.

Reply:
Sure, thank you! I missed a couple of places